docs/docs/workflows/steps/code/auth/README.md

```diff
@@ -1,14 +1,23 @@
 # Auth in code steps
 
```

Comment:
Updated to note that these docs only apply to Node.js code steps, and generally updated the doc to clarify some things.